### DISCUSSION TITLE
How to group components inside a column?

### DISCUSSION THREAD

**Discussion by francoism90** [filamentphp/filament#10305] [Help]
Is it possible to combine multiple components, inside one table column?
```
name || roles
---
image-column (avatar) + text-column (name) || text-column (roles)
```
The `name` should be sortable.

It looks like you can use a different layout (Split, Stack, etc.), however this alters the table layout.

Thanks!

**Comment by leandrocfe** [comment]
Hum, [custom column](https://filamentphp.com/docs/3.x/tables/columns/custom) I think

**Reply by francoism90** [reply]
@leandrocfe Thanks, but after some testing, I think Split is actually a good solution. :)

**Reply by leandrocfe** [reply]
I agree
